Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -54,6 +54,7 @@ protected function configure()
             ->addOption('em', null, InputOption::VALUE_REQUIRED, 'The entity manager to use for this command.')
             ->addOption('shard', null, InputOption::VALUE_REQUIRED, 'The shard connection to use for this command.')
             ->addOption('purge-with-truncate', null, InputOption::VALUE_NONE, 'Purge data by using a database-level TRUNCATE statement')
+            ->addOption('purge-excluded', null, InputOption::VALUE_IS_ARRAY|InputOption::VALUE_OPTIONAL, 'Table/view names to be excleded from purge')
```

Comment:
See https://github.com/doctrine/data-fixtures/pull/350 :)

Reply:
I am not sur what do you want say ? You think it is useless ?

Reply:
No, I just think "excluded" should be spelt properly ;)